pandas/tests/io/excel/test_style.py

```diff
@@ -7,163 +7,158 @@
 from pandas.io.excel import ExcelWriter
 from pandas.io.formats.excel import ExcelFormatter
 
+pytest.importorskip("jinja2")
```

Comment:
this is essentially a new requirement for this right? its ok ,but let's fully document this change.

Reply:
so, actually this file only had one test and that test already had `importskip("jinja2")`, so I haven't added anything, just moved it top level to share across all tests.

I noticed, however, that technically it is would be possible to create a `Styler`, apply methods and then do `.to_excel` all without `jinja2`. The only problem is that `Styler` does a check for `jinja2` at initialisation, so this is not currently functionally possible.

Reply:
its fine to require jinja2 for this, just mention this in the docs / doc-string somewhere. (pretty sure we should be testing this but pls ensure somewhere (and skipping elsewhere)